src/App.tsx

```diff
@@ -1,42 +1,32 @@
-import React from 'react';
+import { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-export const tabs = [
+import Tab from './types/Tab';
+
+import { Tabs } from './components/Tabs';
+
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

Comment:
save `id` not the whole object
`Save the selectedTabId in the App (the first tab is selected by default);`